client/src/views/files/handler/FileHandler.vue

```diff
@@ -452,46 +429,29 @@ async function onClick(event: MouseEvent): Promise<void> {
   }
 }
 
-async function downloadFile(): Promise<void> {
+async function downloadFile(path: FsPath): Promise<void> {
   const workspaceHandle = getWorkspaceHandle();
   if (!workspaceHandle) {
     window.electronAPI.log('error', 'Failed to retrieve workspace handle');
     return;
   }
-  const workspaceInfoResult = await getWorkspaceInfo(workspaceHandle);
-  if (!workspaceInfoResult.ok) {
-    window.electronAPI.log(
-      'error',
-      `Failed to retrieve workspace info: ${workspaceInfoResult.error.tag} (${workspaceInfoResult.error.error})`,
-    );
-    return;
-  }
 
   if (!contentInfo.value) {
     window.electronAPI.log('error', 'No content info when trying to download a file');
     return;
   }
 
-  const config = await storageManager.retrieveConfig();
-  if (!config.disableDownloadWarning) {
-    const { result, noReminder } = await askDownloadConfirmation();
+  try {
```

Comment:
Incorrect conflict resolution, what you're deleting here is the most recent version.